worlds/cccharles/Regions.py

```diff
@@ -0,0 +1,294 @@
+from BaseClasses import MultiWorld, Region, ItemClassification
+from . import CCCharlesItem
```

Comment:
```suggestion
from .Items import CCCharlesItem
```
This is an indirect import that relies on this being imported in `__init__.py`. Best to import it directly.